src/company_gui.cpp

```diff
@@ -1790,13 +1794,27 @@ struct CompanyInfrastructureWindow : Window
 	RailTypes railtypes; ///< Valid railtypes.
 	RoadTypes roadtypes; ///< Valid roadtypes.
 
-	uint total_width; ///< String width of the total cost line.
+	uint      max_label_width; ///< Infrastructure label size.
```

Comment:
We don't try to align comments in new code. Same for the next few lines.

```suggestion
	uint max_label_width; ///< Infrastructure label size.
```